addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayBridgeHandler.java

```diff
@@ -0,0 +1,740 @@
+    public void deleteInstanceResponse(boolean status) {
+    }
+
+    @Override
+    public void getDevicesResponse(DeviceList deviceList) {
+    }
+
+    @Override
+    public void postDeviceResponse(Device device) {
+    }
+
+    @Override
+    public void getDeviceResponse(Device device) {
+    }
+
+    @Override
+    public void getDeviceCommandResponse(String message) {
+    }
+
+    @Override
+    public void getLocationsResponse(LocationList locationList) {
+    }
+
+    @Override
+    public void postLocationResponse(Location location) {
+    }
+
+    @Override
+    public void getLocationResponse(Location location) {
+    }
+
+    @Override
+    public void putLocationResponse(Location location) {
+    }
+
+    @Override
+    public void deleteLocationResponse(boolean status) {
+    }
+
+    @Override
+    public void getProfilesResponse(ProfileList profileList) {
+    }
+
+    @Override
+    public void postProfileResponse(Profile profile) {
+    }
+
+    @Override
+    public void getProfileResponse(Profile profile) {
+    }
+
+    @Override
+    public void putProfileResponse(Profile profile) {
+    }
+
+    @Override
+    public void deleteProfileResponse(boolean status) {
+    }
+
+    @Override
+    public void getNotificationsResponse(NotificationList notificationList) {
+    }
+
+    @Override
+    public void getNotificationResponse(Notification notification) {
+    }
+
+    @Override
+    public void putNotificationResponse(Notification notification) {
+    }
+
+    @Override
+    public void getDeviceHistoriesResponse(DeviceHistoryList deviceHistoryList) {
+    }
+
+    @Override
+    public void getDeviceHistoryResponse(DeviceHistory deviceHistory) {
+    }
+
+    @Override
+    public void apiError(String message, boolean invalidateState) {
+        if (invalidateState) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, message);
+        }
+        logger.warn(message);
+    }
+
+    @Override
+    public void httpStatusError(int httpStatus, String message, boolean invalidateState) {
+        if (invalidateState) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                    message + "(HTTP status code: " + httpStatus + ").");
+        }
+        logger.warn("Z-Way library - {} (HTTP status code: {}).", message, httpStatus);
+    }
+
+    @Override
+    public void authenticationError() {
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                "Authentication error. Please check username and password.");
+        logger.warn("Z-Way library - Authentication error");
+    }
+
+    @Override
+    public void responseFormatError(String message, boolean invalidateApiState) {
+        if (invalidateApiState) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, message);
+        }
+        logger.warn("Z-Way library - Response format error: {}", message);
```

Comment:
reduce the debug - the user is already informed through the Thing status

Or is this indeed a system failure that should be reported?